packages/dd-trace/test/appsec/iast/analyzers/index.spec.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
```

Comment:
I have changed some tests to remove the dependency to datadog-core. but `iast/index.spec.js` continues having these dependencies. 
I have seen 7 test in `integration-tests` directory. Are there more tests in any other directory or project? In order to have some other test for reference.

Reply:
There aren't many right now since we haven't yet started the task of rewriting the core tests, but it should be enough for now if you reduced the mocking to the minimum required.

Reply:
Ok, we are going to open another PR to change test of `packages/dd-trace/test/appsec/iast/analyzers/index.spec.js` by integration-tests